website/blog/2022-02-22-towards-better-releases.md

```diff
@@ -0,0 +1,60 @@
+---
+author: Vadim Chelyshov
+title: Towards better releases
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+Hi, Metals folks!
+
+You may have noticed that the previous `0.11.0` release didn't go smoothly.
+Unfortunately, a group of issues was found only after the release was published.
+Some of them were quite critical. The only option to continue to work was to downgrade Metals until `0.11.1` arrived.
+
+We apologize for that!
+
+However, this post is not only about saying sorry.
+The reasons why it's happened weren't a complete surprise for us.
+At the release moment, we were aware of the weaknesses that we had, and they struck us badly.
```

Comment:
Instead of this would do:
```
In order to remedy it and to avoid such situations in the future we have taken upon us to take steps that will allow us to detect issues early. Some of these issues were quite obvious and we were aware of them for a while.
```

Reply:
replaced with suggested but without `to remedy it` part.